widgetssdk/src/main/res/values/donottranslate.xml

```diff
@@ -3,10 +3,5 @@
 
     <string name="glia_outlined_options_icon_content_description">Outlined options icon description</string>
     <string name="glia_chat_media_upgrade_item_mic_content_description">Timer microphone </string>
```

Comment:
should move those to strings.xml as well?

Reply:
Fixed.
Also, it looks like the OutlinedOptionView is unused.